Defs/Ammo/Pistols/44Magnum.xml

```diff
@@ -99,7 +99,7 @@
 		<label>.44 Magnum bullet (FMJ)</label>
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
 			<damageAmountBase>15</damageAmountBase>
-			<armorPenetrationSharp>8</armorPenetrationSharp>
+			<armorPenetrationSharp>6</armorPenetrationSharp>
```

Comment:
What info is this stat changes based upon?

Reply:
Realism reasons based on gel penetration data and more proper scaling